bool isRunning
---
geometry_msgs/PoseStamped robberGoalResponse
